cross/i686-w64-mingw32-gcc/Portfile

```diff
@@ -22,10 +22,13 @@ depends_lib-append  port:${mingw_target}-headers
 #                   port:cloog
 #                   port:ppl
 
-patchfiles-append   notparallel-install-headers.patch
+patchfiles-append   patch-notparallel-install-headers.diff
 
 # Fixes libgomp compile
-patchfiles-append   0020-libgomp-Don-t-hard-code-MS-printf-attributes.patch
+patchfiles-append   patch-libgomp-Don-t-hard-code-MS-printf-attributes.diff
+
+# Fix ARM64
+patchfiles-append   patch-ARM64.diff
```

Comment:
@kencu: do you have any better idea about how to solve this for all cross-compilers at a single place?

Reply:
This new patch taken from homebrew it’s used in one of there gcc cross-compilers and I’d then reused it for updating there mingw-gcc Formula.

Reply:
@mojca -- not a simple way. As we imagine this will be disappearing again with some upcoming update, it may be the most clear to just carry it in each place, and then delete again.

@Gcenx -- thanks for your ongoing efforts to help us & @mojca  keep all these (tricky) cross compilers in a working state!

Reply:
Changed the note clearing up the origin of the ARM64 patch.